src/cpu/meson.build

```diff
@@ -23,7 +23,7 @@ core_selection = [
     [ ['aarch64'],                ['auto', 'dynrec'],  'C_DYNREC',      'ARMV8LE', 1,         1 ], # ARMv8+ (64-bit)
     [ ['arm'],                    ['auto', 'dynrec'],  'C_DYNREC',      'ARMV7LE', 1,         1 ], # ARMv7+
     [ ['armv6'],                  ['auto', 'dynrec'],  'C_DYNREC',      'ARMV4LE', 0,         0 ], # ARMv6 or older
-    [ ['ppc64',   'powerpc64'],   ['auto', 'dynrec'],  'C_DYNREC',      'PPC64LE', 1,         1 ], # 64 bit PPC processors
+    [ ['ppc64',   'powerpc64'],   ['auto', 'dynrec'],  'C_DYNREC',      'POWERPC', 1,         1 ], # 64 bit PPC processors
```

Comment:
Can't we duplicate POWERPC to `PPC64` or `PPC64BE` to separate the 2 targets?

Reply:
I think if you want more substantial changes, you'll need to do the work yourself as a PPC user 😄

Reply:
Actually, it just dawned on me what's going on. This is in fact specifying the _32-bit_ PowerPC JIT, which is inherently big-endian. But 32-bit instructions will run fine on a 970MP. This is probably the best option for now since I'm still pretty sure the 64-bit JIT isn't necessarily endian-safe, barring any of the weird edge cases we've smoked out of the 64-bit Power dynrec.